polkadot/runtime/common/src/slots/mod.rs

```diff
@@ -221,6 +256,56 @@ pub mod pallet {
 			};
 			Ok(())
 		}
+
+		/// Try to refund the lease deposit before the actual end of the lease.
+		///
+		/// This is only allowed if parachain is in its last lease period and they started with more
+		/// than [`Config::MinLeasePeriodsForEarlyRefund`]. This is useful for parachains who want
+		/// to get access to their funds they used in the last lease and rebid using same for the
+		/// next lease.
+		///
+		/// Can only be called by the Lease Admin or the Parachain manager.
```

Comment:
Looks like `Registrar` and `ForceOrigin`?

Reply:
Registrar is responsible for registration of para_id. We get the manager of para_id from the registrar. 

Good point about Force Origin, I will fix the doc. The ForceOrigin is set to be LeaseAdmin origin in the polkadot and kusama runtimes but from this pallet's POV, it could be any configured origin.